documentation/modules/cruise-control/con-optimization-goals.adoc

```diff
@@ -102,6 +102,7 @@ spec:
       inboundNetwork: 10000KB/s
       outboundNetwork: 10000KB/s
     config:
+      # Note that `hard.goals` must be a subset of `goals` and `default.goals`
```

Comment:
the only thing which is not clear in this example is ... what are the `default.goals`? Are these two hard goals really a subset of them? We know that we are configuring the default ... default goals. Did you check it?